creator-node/src/services/stateMachineManager/stateMachineConstants.js

```diff
@@ -53,5 +53,10 @@ module.exports = {
       key: 'ENTIRE_REPLICA_SET',
       value: 4
     }
+  }),
+  // Describes the type of sync operation
```

Comment:
this isn't yet used anywhere inside stateMachineManager right

Reply:
that's right -- they're only needed in the test file because it calls through to some Snapback code. it works since the "enum" is really just a string